libretro-common/include/libretro.h

```diff
@@ -7311,7 +7311,18 @@ struct retro_device_power
     */
    int8_t percent;
 };
-
+enum retro_sensor_selector
+{
+   RETRO_SENSOR_ACCELEROMETER_X,
+   RETRO_SENSOR_ACCELEROMETER_Y,
+   RETRO_SENSOR_ACCELEROMETER_Z,
+   RETRO_SENSOR_GYROSCOPE_X,
+   RETRO_SENSOR_GYROSCOPE_Y,
+   RETRO_SENSOR_GYROSCOPE_Z,
+   RETRO_SENSOR_ILLUMINANCE,
+   RETRO_SENSOR_MAX,
+   RETRO_SENSOR_SELECTOR_DUMMY=INT_MAX
+};
```

Comment:
Couple of remarks here:

- Could you move this back to where the original equivalent `#define`s were?
- Could you restore the doc comments that were there before you changed this?

Reply:
Done